### PR TITLE
Create slot time side menu

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/routes/bookAVisit.test.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ describe('GET /book-a-visit/select-visitors', () => {
       },
     }
 
-    prisonerVisitorsService.getVisitors.mockResolvedValue(returnData)
+    prisonerVisitorsService.getApprovedVisitors.mockResolvedValue(returnData)
```
Comment (psoleckimoj) [inline server/routes/bookAVisit.test.ts:154]: Renamed this to make it match what it actually does. Could've added an arg to the function but it's obvious what it does when the function is named like this.

File: server/@types/bapv.d.ts [hunk 2]
```diff
@@ -176,7 +176,7 @@ export type VisitSessionData = {
     location: string
   }
   visit?: VisitSlot
-  visitRestriction?: 'OPEN' | 'CLOSED'
+  visitRestriction?: 'OPEN' | 'CLOSED' | 'UNKNOWN'
```
Comment (tpmcgowan) [inline server/@types/bapv.d.ts:179]: I guess the `UNKNOWN` is needed to show migrated visits that may not have a type set?

File: server/services/prisonerVisitorsService.ts [hunk 3]
```diff
@@ -25,4 +25,18 @@ export default class PrisonerVisitorsService {
 
     return visitorList
   }
+
+  async getAllVisitors(offenderNo: string, username: string): Promise<VisitorListItem[]> {
```
Comment (tpmcgowan) [inline server/services/prisonerVisitorsService.ts:29]: Might be missing something, but is this new `getAllVisitors()` actually used?

File: server/@types/bapv.d.ts (hunk 2 above)
Comment (psoleckimoj) [reply server/@types/bapv.d.ts:179]: It was moaning at me that it was in the pulled in types.

**Pull request merged**